diff -urNp -x '*.orig' phonon-4.10.3.org/phonon.pc.cmake phonon-4.10.3/phonon.pc.cmake
--- phonon-4.10.3.org/phonon.pc.cmake	2019-07-19 13:42:16.000000000 +0200
+++ phonon-4.10.3/phonon.pc.cmake	2022-05-15 16:30:58.301489764 +0200
@@ -1,7 +1,7 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=@EXEC_INSTALL_PREFIX@
 libdir=@ABS_LIB_INSTALL_DIR@
-includedir=@ABS_INCLUDE_INSTALL_DIR@
+includedir=@ABS_INCLUDE_INSTALL_DIR@/KDE
 buildsystemdir=@ABS_BUILDSYSTEM_INSTALL_DIR@
 
 Name: @PHONON_LIB_SONAME_CAMEL@
